cli/localci/core/boost_cache.py

```
@@ -0,0 +1,100 @@
+        return
+
+    _git_fetch_and_update(boost_dir, branch, shallow)
+
+
+def _git_clone(dest: Path, branch: str, shallow: bool, remote_url: str) -> None:
+    args = ["git", "clone", "--branch", branch]
+    if shallow:
+        args.extend(["--depth", "1"])
+    args.extend([remote_url, str(dest)])
+    try:
+        subprocess.run(args, check=True, capture_output=True, text=True)
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)